content/docs/06-integrations/01-maintenance-policy.md

```diff
@@ -133,26 +133,33 @@ We provide add-on pack updates as follows:
 ## Packs Deprecation
 
 
-<!-- We deprecate and remove packs when a more stable version of the pack is available or when the underlying technology becomes obsolete. -->
+We deprecate and remove packs when a more stable version of the pack is available or when the underlying technology becomes obsolete. When a pack is deprecated, you will still be able to create new cluster profiles using the pack and deploy clusters that use profiles containing the pack. Our Release Notes announce when a pack is deprecated and a replacement pack is available.
```

Comment:
```suggestion
We deprecate and remove packs when a more stable version of the pack is available or when the underlying technology becomes obsolete. When a pack is deprecated, you will still be able to create new cluster profiles using the pack and deploy clusters that use profiles containing the pack.
```
I removed the Release Notes statement because we are not doing that.